cli/root.go

```diff
@@ -992,7 +992,7 @@ func DumpHandler(ctx context.Context) {
 		if err != nil {
 			dir = os.TempDir()
 		}
-		fpath := filepath.Join(dir, fmt.Sprintf("coder-agent-%s.dump", time.Now().Format("2006-01-02T15:04:05.000Z")))
+		fpath := filepath.Join(dir, fmt.Sprintf("coder-%s-%s.dump", name, time.Now().Format("2006-01-02T15:04:05.000Z")))
```

Comment:
out of scope: we should probably not use colons in the file path as that's a forbidden character in NTFS.

Reply:
Good catch, it's such a small change I could piggy back it on this one.